mkdocs.yml

```diff
@@ -15,6 +15,7 @@ nav:
       - Download: "setup/download.md"
       - Installation: "setup/installation.md"
       - Configuration: "setup/configuration.md"
+      - Custom_cards: "setup/custom_cards.md"
```

Comment:
@stokkie90 Is this the only change I need to do to create a new page?

Reply:
The key in this case is the Title. So i would change it to: `"Custom Cards": "setup/custom-cards.md"`